### ISSUE TITLE
Don't Pos. Named Arguments as headers

### ISSUE THREAD

**Issue by mmagnus** [ribozz/sphinx-argparse#78]
I want to update sphinx-argparse to be able to use `:nodescription:`. 

However, when I update the package I get something like this:

![screen shot 2017-09-04 at 3 18 26 pm](https://user-images.githubusercontent.com/118740/30028176-5623e600-9184-11e7-8b7b-c2a289640d97.png)

is there a way to get rid of these arguments, and keep it as it was before?

Like here https://rna-pdb-tools.readthedocs.io/en/latest/utils.html#cluster 

BTW Thanks for this great tool!

**Comment by dpryan79** [comment]
Try setting the depth for table of contents:

    .. toctree::
       :maxdepth: 2

In newer versions all of the sections are linkable, whereas they weren't in old versions. A side-effect of linkability is possible inclusion in the table of contents.

**Comment by mmagnus** [comment]
@dpryan79 yeah, thanks. I was thinking of that, but I have different levels.

![screen shot 2017-09-04 at 3 49 18 pm](https://user-images.githubusercontent.com/118740/30029222-ad9c031e-9188-11e7-8b9d-71b7f0e59399.png)

Do you think, that there is another way to stop these inclusions?

**Comment by dpryan79** [comment]
Not to my knowledge, but I'll have to look into it.

**Comment by mmagnus** [comment]
Thanks!

Eh, I don't know the code. It would be great to add a directive to enable or disable it. Or maybe it's just me ;-)

**Comment by tgarc** [comment]
I have the same issue. I don't want to add these extra links to the table of contents. Is there a way to control what heading level the headings will be on?

**Comment by mmagnus** [comment]
... and I still think it would be nice to be able to on/off this feature (which is pretty nice, but not always that is what we want).

**Comment by mmagnus** [comment]
Hey, there. Is there any way to control this behavior right now? This is the thing that keeps me from updating the library in my project.

**Comment by dpryan79** [comment]
@mmagnus Not at the moment. I'm pretty slow when it comes to dealing with issues on this package, but if you want to make a PR I'll happily merge it and post a new release :)
